store/splus.js

```diff
@@ -6,7 +6,16 @@ import * as chroma from 'chroma-js';
 
 export const state = () => ({
   schedule: undefined,
-  schedules: SCHEDULES,
+  schedules: SCHEDULES.map(
+    (schedule) => ({ ...schedule, path: `${schedule.faculty} ${schedule.degree}` })),
```

Comment:
`path` wurde in mehreren Komponenten berechnet. Erst habe ich es in eine Bibliothek verlagert - aber an die Quelle passt es noch besser.